Extensions/PathfindingBehavior/tests/commonpathfindingruntimebehavior.spec.js

```diff
@@ -0,0 +1,128 @@
+
+const addObstacle = runtimeScene => {
+  var obstacle = new gdjs.RuntimeObject(runtimeScene, {
+    name: 'obstacle',
+    type: '',
+    behaviors: [
+      {
+        type: 'PathfindingBehavior::PathfindingObstacleBehavior',
+        impassable: true,
+        cost: 2
+      },
+    ],
+  });
+  obstacle.getWidth = function() {
+    return 100;
+  };
+  obstacle.getHeight = function() {
+    return 100;
+  };
+  runtimeScene.addObject(obstacle);
+
+  return obstacle;
+};
+
+// tests cases where every collisionMethod has the same behavior.
+describe('gdjs.PathfindingRuntimeBehavior', function() {
+["Legacy", "AABB", "HitBoxes"].forEach((collisionMethod) => {
+describe(`(collisionMethod: ${collisionMethod})`, function() {
```

Comment:
Nitpicking: Run Prettier (I installed the extension in VS Code, should also be available for other IDEs) on the file (and basically any JS/TS file) to auto format the tests.

Reply:
Formatters are important to have clean commits, but I prefer to do it right before the merge because they make it difficult for me to read the code. But I'm open to negotiations...